packages/rs-sdk/src/core.rs

```diff
@@ -0,0 +1 @@
+//! Core related definitions.
```

Comment:
maybe remove that empty file?

Reply:
Idk, maybe we can remove the whole crate since it practically does nothing